auth-web/src/util/constants.ts

```diff
@@ -248,6 +248,13 @@ export enum NrEntityType {
     INFO = 'INFO', // special value for sub-menu
 }
 
+export enum NrRequestActionStrings {
+  // Can add other NrRequestActionCodes here to use the action code instead of the NrRequestTypeCd.
+  // Must ensure that the action code does not have several potential types
+  // Example: the NEW action code can be for Incorporation or Registration, so we cannot use it for the NR type
```

Comment:
I like the caveat but this further indicates that this should be all in common utils, where proper logic can be applied and no mistake can be made by dereferencing the array by "NEW".